src/cache.cpp

```diff
@@ -209,19 +207,27 @@ namespace {
 	}
 }
 
-#define macro(r, data, elem) \
+#define cache(elem) \
 	BitmapRef Cache::elem(const std::string& f) { \
 		bool trans = spec[Material::elem].transparent; \
 		return LoadBitmap<Material::elem>(f, trans); \
 	}
-
-BOOST_PP_SEQ_FOR_EACH(macro, ,
-					  (Backdrop)(Battle)(Battle2)(Battlecharset)(Battleweapon)
-					  (Charset)(Chipset)(Faceset)(Gameover)(Monster)
-					  (Panorama)(System2)(Frame)(Title)(System)
-					  )
-
-#undef macro
+	cache(Backdrop)
+	cache(Battle)
+	cache(Battle2)
+	cache(Battlecharset)
+	cache(Battleweapon)
+	cache(Charset)
+	cache(Chipset)
+	cache(Faceset)
+	cache(Gameover)
+	cache(Monster)
+	cache(Panorama)
+	cache(System2)
+	cache(Frame)
+	cache(Title)
+	cache(System)
```

Comment:
Imo this could be even replaced with real function definitions. Is redundant but more clear then this (or that ugly boost pp). What do you think?